client/src/hooks/useMaterialsSearch.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useSearch } from './useSearch'
+import { ResourcesPortalContext } from '../ResouresPortalContext'
+
+export const useMaterialsSearch = (newSearch) => {
```

Comment:
rename this to `useResourcePortalSearch` since this uses the `ResourcePortalContext` (to be consistent with other searches that aren't "global")

Reply:
changed to `useSearchResources`